### PR TITLE
Add warning for installed PTVS + initial work on single VSIX

### PR DESCRIPTION
**Pull request by MikhailArkhipov**
- Add warning for installed PTVS at Pylance LSC activation (the safest place I could find).
- In order to coexist with PTVS (since we can't block installation) changes in GUIDs are required, leaving all legacy guids behind.
- Merge Profiler and Cookiecutter into Pylance VSIX

### CODE REVIEW

File: Python/Product/PythonTools/PythonToolsPackage.cs [hunk 1]
```diff
@@ -78,15 +78,15 @@ namespace Microsoft.PythonTools {
...
     [ProvideLanguageExtension(typeof(PythonLanguageInfo), PythonConstants.FileExtension)]
     [ProvideLanguageExtension(typeof(PythonLanguageInfo), PythonConstants.WindowsFileExtension)]
     [ProvideLanguageExtension(typeof(PythonLanguageInfo), PythonConstants.StubFileExtension)]
     [ProvideDebugEngine(AD7Engine.DebugEngineName, typeof(AD7ProgramProvider), typeof(AD7Engine), AD7Engine.DebugEngineId, hitCountBp: true)]
     [ProvideDebugAdapter("VSCode Python Debugger", DebugAdapterLauncher.VSCodeDebugEngineId, DebugAdapterLauncher.DebugAdapterLauncherCLSID, CustomDebugAdapterProtocolExtension.CustomProtocolExtensionCLSID, "Python", "{DA3C7D59-F9E4-4697-BEE7-3A0703AF6BFF}", typeof(DebugAdapterLauncher), typeof(CustomDebugAdapterProtocolExtension))]
     [ProvideDebugLanguage("Python", "{DA3C7D59-F9E4-4697-BEE7-3A0703AF6BFF}", PythonExpressionEvaluatorGuid, AD7Engine.DebugEngineId)]
-    [ProvideDebugPortSupplier("Python remote (ptvsd)", typeof(PythonRemoteDebugPortSupplier), PythonRemoteDebugPortSupplier.PortSupplierId, typeof(PythonRemoteDebugPortPicker))]
```
Comment (huguesv) [inline Python/Product/PythonTools/PythonToolsPackage.cs:89]: This shouldn't be commented out. It provides the "Python Remote (debugpy)" entry in debug/attach combo box ie. it's not only for legacy debugger. I just verified this.

File: Python/Product/Profiling/Profiling.csproj [hunk 2]
```diff
@@ -165,27 +142,16 @@
   </ItemGroup>
   <ItemGroup>
     <Content Include="proflaun.py">
-      <IncludeInVSIX>true</IncludeInVSIX>
```
Comment (huguesv) [inline Python/Product/Profiling/Profiling.csproj:168]: Does this file get included in the pylance.vsix now? I'll try it out later this afternoon.

File: Python/Product/PythonTools/PythonToolsPackage.cs (hunk 1 above)
Comment (MikhailArkhipov) [reply Python/Product/PythonTools/PythonToolsPackage.cs:89]: OK, the name was confusing...

File: Python/Product/Profiling/Profiling.csproj (hunk 2 above)
Comment (MikhailArkhipov) [reply Python/Product/Profiling/Profiling.csproj:168]: Yes, we need to check if all files that are not assemblies are included... I may have to update VSIX for Pylance manually.

**Comment by huguesv** [comment]
I'm getting a build error when I pulled from your branch:
```
41>  Pylance -> C:\Projects\GitHub\huguesv\PTVS\BuildOutput\Debug16.0\raw\binaries\Microsoft.PythonTools.Pylance.dll
41>C:\Projects\GitHub\huguesv\PTVS\packages\Microsoft.VSSDK.BuildTools\tools\vssdk\Microsoft.VsSDK.targets(1140,5): error : CreatePkgDef : error : ProvideCodeBaseAttribute: Could not load specified assembly: 'Microsoft.PythonTools.ProjectWizards' reason: Could not load file or assembly 'file:///C:\Projects\GitHub\huguesv\PTVS\BuildOutput\Debug16.0\raw\binaries\Microsoft.PythonTools.ProjectWizards.dll' or one of its dependencies. The system cannot find the file specified.
41>C:\Projects\GitHub\huguesv\PTVS\packages\Microsoft.VSSDK.BuildTools\tools\vssdk\Microsoft.VsSDK.targets(1140,5): error : 
```

ProjectWizards seem to be gone from the solution.

File: Python/Product/Profiling/Profiling.csproj (hunk 2 above)
Comment (MikhailArkhipov) [reply Python/Product/Profiling/Profiling.csproj:168]: Let me try chaining. According to docs we may be able to keep multiple VSIX and instead nest them in Pylance vsix
https://docs.microsoft.com/en-us/visualstudio/extensibility/vsix-extension-schema-2-0-reference?view=vs-2019

**_Location** - This optional attribute specifies either the relative path within this VSIX to a nested VSIX package or a URL to the download location for the dependency. This attribute is used to help the user locate the prerequisite package._

**Comment by MikhailArkhipov** [comment]
Will reopen from another branch

**Pull request closed without merging**